### DISCUSSION TITLE
Pylontech US3000C & 2x Mppt 100/20

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#1345] [General]
Hallo Leute,

habe die Anlage meines Nachbarn mit der Platine von @swingstate in Betrieb genommen.
Seid ihr so freundlich, mal über die Settings zu schauen?
Meiner Meinung nach wird die Batterie nicht richtig voll geladen.
Sie ist bei 60% stehen geblieben, obwohl noch genügend PV-Leistung verfügbar war.
Die Einstellung der beiden MPPTs sind identisch.

LG Robert

![1](https://github.com/user-attachments/assets/660ea6a4-7844-4957-b7c5-cec1c2abc9dd)
![OpenDTU-OnBattery (1)](https://github.com/user-attachments/assets/8eda8c88-e32c-4557-9883-9f196857c714)
![OpenDTU-OnBattery](https://github.com/user-attachments/assets/45ada0cf-6e13-4705-998c-cae39ff52bec)
![Screenshot_20241020_153842](https://github.com/user-attachments/assets/991a7a30-db24-401b-9744-a44c11cd5f82)

**Comment by schlimmchen** [comment]
> Sie ist bei 60% stehen geblieben, obwohl noch genügend PV-Leistung verfügbar war.

![image](https://github.com/user-attachments/assets/39a480d2-e205-44e4-8775-578089e53bdb)

Das sieht mir falsch aus und würde erklären, dass die Batterie nicht voll wird. Diese Pylontech haben doch 16 Zellen, oder sogar 17? Demnach wäre ein sinnvoller Wert für die Absobtionsspannung vermutlich etwas in der Größenordnung um 55.5V.

Jemand mit einer Pylontech Batterie kann das aber genauer sagen. Vielleicht weiß @Manos1966 das?

**Reply by Manos1966** [reply]
Guten Morgen Bernhard,
15 Zellen

**Reply by Manos1966** [reply]
Unter 
https://github.com/hoylabs/OpenDTU-OnBattery/wiki/Battery%E2%80%90Tipps

Unter Pylontech, ist ein Bild mit dem Spannungswerten der Pylontech Batterien.

**Reply by Sorbat** [reply]
> Spannungswerten der Pylontech Batterien.

Gibt es sowas vielleicht auch für die Pytes?

**Reply by Manos1966** [reply]
Full Solar Passthrough bei 66% ?
Willst du nicht lieber 85% stellen (wir haben Winter)

**Reply by Manos1966** [reply]
Entladestromlimit 37A ? 
Ist es eine US2500?

Hast du deine Victron MPPT Einstellungen angepasst (max 52,4V)?

Wir haben den Link für das Victron Papier für Pylontech Batterien in das Wiki.

**Reply by schlimmchen** [reply]
> Full Solar Passthrough bei 66% ?

66V, nicht 66%.

**Reply by knopers1** [reply]
es ist doch eine US3000 - 15 Zellen

**Reply by Manos1966** [reply]
> 66V, nicht 66%.

🤦‍♂️  

Ich hasse Posts auf dem Handy Bildschirm lesen.

**Comment by rs3hc** [comment]
Danke für eure Infos.

Habe die Ladespannung mal auf 53V gesetzt und die Werte in der DTU angepasst.
Ich gebe nochmal Rückmeldung.

![OpenDTU-OnBattery (1)](https://github.com/user-attachments/assets/30179b91-2de3-49e9-84f6-bbde5616324c)
![OpenDTU-OnBattery](https://github.com/user-attachments/assets/c0a42d79-f302-4f5c-82df-285249966845)
![Screenshot_20241025_080357](https://github.com/user-attachments/assets/b09919b7-3fb7-48dc-95bd-f9f7a2031a65)
